cpp/include/raft/util/bitonic_sort.cuh

```diff
@@ -173,10 +173,10 @@ class bitonic {
   friend class bitonic;
 
   template <typename KeyT, typename... PayloadTs>
-  static __device__ __forceinline__ void merge_(bool ascending,
-                                                int warp_width,
-                                                KeyT* __restrict__ keys,
-                                                PayloadTs* __restrict__... payloads)
+  static __device__ __forceinline__ void merge_impl(bool ascending,
```

Comment:
Can you use `RAFT_DEVICE_INLINE` (or define it alongside `RAFT_HOST_DEVICE_INLINE` if it doesn't exist?